composer.json

```diff
@@ -67,7 +67,10 @@
     "autoload": {
         "psr-4": {
             "Mezzio\\Swoole\\": "src/"
-        }
+        },
+	"files": [
+            "compat/openswoole.php"
+        ],
```

Comment:
This trailing comma is currently breaking CI! Also, the lock file will need a refresh too